doc/tutorials/transformer/model_retrieval.ipynb

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        # Maybe download the model from Huggingface hub
</pre><p>Maybe sounds uncertain. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>That's meant, as the downloading will only take place if the model is not found in the cache.</p>

Reply:
ok then detail it in the comment